src/main/java/io/confluent/kafka/connect/datagen/ConfigUtils.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.kafka.connect.datagen;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Parser;
+import org.apache.avro.SchemaParseException;
+import org.apache.kafka.common.config.ConfigException;
+
+public class ConfigUtils {
+  public static Schema getSchemaFromSchemaString(String schemaString) {
+    Schema.Parser schemaParser = new Parser();
+    Schema schema;
+    try {
+      schema = schemaParser.parse(schemaString);
+    } catch (SchemaParseException e) {
+      throw new ConfigException("Unable to parse the provided schema");
```

Comment:
Small nit: it'd be nice to see the full stack trace here. We definitely don't want to use the two-arg constructor for `ConfigException` since that's meant for a key/value pair and not a message/exception pair. Do you think we can log these errors before throwing our own `ConfigException`?

Reply:
Thanks, I think this was definitely needed.